view/frontend/templates/bottomline.phtml

```diff
@@ -4,16 +4,18 @@
     div.yotpo.bottomLine { display: inline-block; margin: 0 10px 1px 0;}
     .thumbnail .yotpo a { display: inline-block; }
     div.yotpo.QABottomLine { display: inline-block; }
+    .product-info-main .product-reviews-summary{width:100%;}
 </style>
 <?php } ?>
+<?php if( !$this->isRenderCategoryBottomline()) { ?>
```

Comment:
what is the logic behind this if statement? when the category bottomline is enabled It ignores the status of the product page bottomline that defined in the configuration page.

Check this case in configuration page : 

Star Ratings On Category Pages --> Yes (enabled)
Show bottomline --> No (disabled)

Result: In product page the bottomline still exist.

The category bottomline status effects both on category pages and product pages,  but needs to be separated.